mmdet/apis/train.py

```diff
@@ -68,6 +68,49 @@ def set_random_seed(seed, deterministic=False):
         torch.backends.cudnn.benchmark = False
 
 
+def scale_lr(cfg, logger):
+    """Automatically scaling LR according to GPU number and sample per GPU.
+
+    Args:
+        cfg (config): training config.
+        logger (logging.Logger): logger.
+    """
+
+    if cfg.enable_auto_scale_lr is False:
+        logger.info(f'You disabled automatically scaling LR.')
+        return
+
+    if "mmdet_official_special_batch_size" in cfg:
+        original_batch_size = cfg.mmdet_official_special_batch_size
+    else:
+        original_batch_size = cfg.default_batch_size
```

Comment:
`enable_auto_scale_lr` and `default_batch_size` don't definitely exists in cfg. We need to use `get`

Reply:
Done